torch/_logging/_internal.py

```diff
@@ -154,14 +154,14 @@ def clear(self):
 
 # sample usage: torch._logging.set_logs(**torch._logging.DEFAULT_LOGGING)
 DEFAULT_LOGGING = {
-    "dynamo": logging.INFO,
-    "graph_code": True,
-    "aot": logging.INFO,
+    "dynamo": logging.DEBUG,
+    "aot": logging.DEBUG,
+    "inductor": logging.DEBUG,
+    "ddp_graphs": True,
     "graph_breaks": True,
+    "guards": True,
     "recompiles": True,
     "dynamic": logging.INFO,
-    "guards": True,
-    "trace_source": True,
```

Comment:
I'm not familiar with `trace_source` - are you taking it out because it's covered by dynamo's debug logging?